CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -22,5 +22,8 @@ public class Employee
         [ForeignKey(nameof(Company))]
         public Guid CompanyId { get; set; }
```

Comment:
Avoid committing TODOs or unfinished code in production files. Temporary additions like debugging or incomplete fields should be removed before merging. This field seems to be for testing and violates the project's clean architecture and domain model purity.

```suggestion
// public string salary;  // Employee compensation - stores amount
```